inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/table/JdbcDialects.java

```diff
@@ -44,6 +45,7 @@ public final class JdbcDialects {
         DIALECTS.add(new TDSQLPostgresDialect());
         DIALECTS.add(new SqlServerDialect());
         DIALECTS.add(new OracleDialect());
+        DIALECTS.add(new PostgresDialect());
```

Comment:
move PostgresDialect(). Because pg use `dialectImpl` to register custom dialect.
see this file JdbcDynamicTableFactory.java
```java
final Optional<JdbcDialect> dialect = dialectImplOptional.map(JdbcDialects::register)
                .orElseGet(() -> JdbcDialects.get(jdbcUrl));
```

Reply:
DONE